src/EnergyPlus/WaterCoils.cc

```diff
@@ -1783,6 +1878,8 @@ namespace WaterCoils {
 				SizingString = WaterCoilNumericFields( CoilNum ).FieldNames( FieldNum ) + " [C]";
 				DataFlowUsedForSizing = DataAirFlowUsedForSizing; // used by air loop coils
 				TempSize = WaterCoil( CoilNum ).DesInletAirTemp;
+				TempSize = AutoSize;
```

Comment:
This erases the user input for design inlet air temp. 2 files shows diff's in eio because of this.

5ZoneAirCooled_ZoneAirMassFlowBalance:

    Component Sizing Information, Coil:Cooling:Water, MAIN COOLING COIL 1, Design Size Design Inlet Air Temperature [C], 21.00753 
    - Component Sizing Information, Coil:Cooling:Water, MAIN COOLING COIL 1, User-Specified Design Inlet Air Temperature [C], 21.60000 
    Component Sizing Information, Coil:Cooling:Water, MAIN COOLING COIL 1, Design Size Design Inlet Water Temperature [C], 7.00000 
    - Component Sizing Information, Coil:Cooling:Water, MAIN COOLING COIL 1, User-Specified Design Inlet Water Temperature [C], 7.00000

    Coil:Cooling:Water,
      Main Cooling Coil 1,     !- Name
      CoolingCoilAvailSched,   !- Availability Schedule Name
      autosize,                !- Design Water Flow Rate {m3/s}
      autosize,                !- Design Air Flow Rate {m3/s}
      7.0,                     !- Design Inlet Water Temperature {C}
      21.6,                    !- Design Inlet Air Temperature {C}

I suggest a followup issue to correct any last minute findings.